frontend/src/metabase/dashboard/components/ClickBehaviorSidebar/ClickBehaviorSidebarHeader/ClickBehaviorSidebarHeader.tsx

```diff
@@ -29,29 +29,30 @@ interface Props {
   onUnsetColumn: () => void;
 }
 
-function ClickBehaviorSidebarHeader({
-  dashcard,
-  selectedColumn,
-  onUnsetColumn,
-}: Props) {
-  const renderContent = useCallback(() => {
-    if (isTableDisplay(dashcard)) {
-      if (selectedColumn) {
-        return (
-          <ColumnClickBehaviorHeader onClick={onUnsetColumn}>
-            <ChevronIconContainer>
-              <Icon name="chevronleft" size={12} />
-            </ChevronIconContainer>
-            <DefaultHeader>{selectedColumn.display_name}</DefaultHeader>
-          </ColumnClickBehaviorHeader>
-        );
-      }
+function HeaderContent({ dashcard, selectedColumn, onUnsetColumn }: Props) {
+  if (isTableDisplay(dashcard)) {
+    if (!selectedColumn) {
```

Comment:
nit^10 

Maybe we can go with `selectedColumn` truthy instead of checking for falsy.

I see that in the current way we return a one-liner first, in this sense it's better.
At the same time, for new readers, holding the false condition in mental RAM is a bit more expensive.